packages/amplify-cli/src/domain/amplify-usageData/FlowReport.ts

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable spellcheck/spell-checker */
+import { getAmplifyLogger, Redactor } from '@aws-amplify/amplify-cli-logger';
+import { IAmplifyLogger } from '@aws-amplify/amplify-cli-logger/lib/IAmplifyLogger';
```

Comment:
same here.

Reply:
fixing this and running verify-api-extract:clean should fix both